test/clonevolume_tester.py

```diff
@@ -379,6 +381,12 @@ def setup_mock_objects(self):
         mock_3parclient.getCPG.return_value = {}
         mock_3parclient.getVolumeMetaData.return_value = {'value': True}
         mock_3parclient.getTask.return_value = {'status': data.TASK_DONE}
+        config = create_configuration('ISCSI')
+        config.hpe3par_iscsi_chap_enabled = True
+        config.use_multipath = False
+        mock_orchestrator = self.mock_objects['mock_orchestrator']
+        mock_orchestrator.return_value = \
+            {'DEFAULT': mgr.VolumeManager(config, config)}
```

Comment:
Couple of general comments (not specific to this section of code):
1. Implement a test case that can exercise two different backends
2. Implement a negative test case by passing non-existing backend

Reply:
Will be done in phase 2 of this implementation.